Documentation/op-guide/grpc_proxy.md

```diff
@@ -64,6 +64,34 @@ Start the etcd gRPC proxy to use these static endpoints with the command:
 $ etcd grpc-proxy start --endpoints=infra0.example.com,infra1.example.com,infra2.example.com --listen-addr=127.0.0.1:2379
 ```
 
+Optionally register proxy with user-provided prefix
+
+```bash
+$ etcd grpc-proxy start --endpoints... \
+  --listen-addr=10.240.0.1:23790 \
+  --advertise-client-url=10.240.0.1:23790 \
+  --resolver-prefix=___grpc_proxy_endpoint \
+  --resolver-ttl=60
+```
+
+Now clients can dynamically discover all proxy endpoints:
```

Comment:
The proxy will list all its members for `member list`:

 ETCDCTL_API=3 ./bin/etcdctl --endpoints=http://10.240.0.1:23790 member list
(show output)

This lets clients automatically discover proxy endpoints through `Sync`: